airflow/www/static/js/dag/details/taskInstance/ExtraLinks.tsx

```diff
@@ -57,8 +57,27 @@ const ExtraLinks = ({
     if (!url) {
       return true;
     }
-    const urlRegex = /^(https?:)/i;
-    return urlRegex.test(url);
+    try {
+      const path = new URL(url);
+      // The URL creation will succeed for
+      // 1. Absolute URL
+      // 2. URL with javascript:()
+      // URL(url) will fail for relative paths.
+      // eslint-disable-next-line
+      if (path.protocol === "javascript:") {
```

Comment:
We likely don't need to check it here - since only http/https are allowed below.

Reply:
Yes, I have updated the code PTAL.